mergify_engine/debug.py

```diff
@@ -189,21 +194,37 @@ def _url_parser(
     )
 
 
+async def get_owner_id_from_login(owner_login: str) -> int:
```

Comment:
You should returns `github_types.GitHubAccountIdType` instead of int.

Reply:
I just realize this method is used in other parts of the engine (web/root.py), so maybe clients/github.py is a better location for this method, to not import the debug module everywhere.